packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -216,11 +221,11 @@ export class DefaultCrudRepository<T extends Entity, ID>
     return this.toEntity(model);
   }
 
-  update(entity: T, options?: Options): Promise<boolean> {
+  update(entity: T, options?: Options): Promise<void> {
```

Comment:
I think it should call `return await this.updateById(entity.getId(), entity, options)`, same for the next function `delete`.

Reply:
It's not necessary.